hello_world.pot

```diff
@@ -0,0 +1,33 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
pot file should not be part of git, as it is dynamically generated.

Reply:
removed from commit